pytest-selenium/tests/test_attribution.py

```diff
@@ -0,0 +1,39 @@
+from selenium import webdriver
+import unittest
+from time import sleep
+
+from selenium.common.exceptions import StaleElementReferenceException
+
+
+class Citationtexturl(unittest.TestCase):
+    def setUp(self):
+        self.browser = webdriver.Chrome()
+        self.browser.implicitly_wait(3)
+
+    def teardown(self):
+        self.browser.quit()
+
+    # c476303 attribution section is initially collapsed, expands when clicked
+    def test_c476303_attribution_initial_status(self):
+        self.browser.get("https://rex-web.herokuapp.com/books/college-physics/pages/preface")
+        sleep(2)
+        self.browser.find_element_by_xpath(
+            '//*[@id="root"]/div[2]/div[4]/div/div[2]/div/div/details'
+        ).click()
+        # y = self.browser.get_attribute(x)
+        # print(y)
+
+        # self.assertEqual(self.browser.get_attribute(details),open)
+
+        # url = 'http://localhost:8000/analyse/'
+
+        org = self.browser.find_element_by_xpath(
+            '//*[@id="root"]/div[2]/div[4]/div/div[2]/div/div/details'
+        )
+        # Find the value of org?
+        # val = org.get_attribute("class")
+        y = org.is_selected()
```

Comment:
Same thing goes here. It's hard to know what `y` is. I know you are playing around with selenium trying to figure things out. That's ok but it's still helpful to name variables so they convey their intent.

`is_org_selected = org.is_selected()` might be better.